src/commands/createNewProject/ProjectCreateStep/PowerShellProjectCreateStep.ts

```diff
@@ -54,9 +73,95 @@ export class PowerShellProjectCreateStep extends ScriptProjectCreateStep {
             await fse.writeFile(profileps1Path, profileps1);
         }
 
+        const majorVersion: number | undefined = await this.getLatestAzModuleMajorVersion();
+        if (majorVersion !== undefined) {
+            progress.report({
+                message: `Successfully retrieved ${this.azModuleName} information from PowerShell Gallery"`,
+                increment: 1
+            });
+        } else {
+            progress.report({
+                message: `Failed to retrieve ${this.azModuleName} information from PowerShell Gallery, creating offline template"`,
```

Comment:
Is this message shown to the user? If it is, could you please change it the following?

`Failed to get Az module version. Edit the requirements.psd1 file when the powershellgallery.com is accessible.`